java/core/src/main/java/sleeper/core/statestore/FileInfo.java

```diff
@@ -35,14 +35,18 @@ public enum FileStatus {
     private final FileStatus fileStatus;
     private final String jobId;
     private final Long lastStateStoreUpdateTime; // The latest time (in milliseconds since the epoch) that the status of the file was updated in the StateStore
+    private boolean countApproximate;
+    private boolean onlyContainsDataForThisPartition;
```

Comment:
We can assert that they're set correctly after ingest, bulk import and each type of compaction. I'd avoid asserting them in every test, but we can ensure they're asserted in one test for each piece of code that sets them.

It also looks like these are set incorrectly in most tests, eg. ones that use `FileInfoFactory`, and everywhere the tests call `FileInfo.builder`. It might be better to use `FileInfoFactory` for most tests.